content/blog/mlops-–-deploying-an-ml-model-in-greenlake-platform-mlops-service.md

```diff
@@ -0,0 +1,318 @@
+---
+title: ML Ops – Deploying an ML model in HPE GreenLake Platform ML Ops service
+date: 2022-08-08T07:48:49.530Z
+author: Thirukkannan M
+authorimage: /img/Avatar1.svg
+tags:
+  - hpe-greenlake
+---
+**Overview**
+
+HPE GreenLake Central is an advanced software-as-a-service platform that provides you with a consistent cloud experience for all your applications and data—on-premises or off-premises. It provides you with insights and controls to manage your hybrid IT estate, complementing your use of public clouds and data centers. HPE GreenLake Central gives you the ability to choose where and how to place your workloads and data, and—through the services you purchase—enables you to monitor security, capacity, resource utilization, and costs.
+
+HPE GreenLake for ML Ops is an on-premises, enterprise-grade ML service, enabling developers and data scientists to rapidly build, train, and deploy ML models—from pilot to production, at any scale.
+
+This preconfigured solution comprises an optimized hardware stack and is powered by HPE Ezmeral Runtime Enterprise. It provides data scientists with self-service access to a sandbox environment for prototyping and testing, to eliminate IT provisioning delays, ensure repeatability, and accelerate time-to-value. As a fully managed solution, the HPE GreenLake for ML Ops offering frees IT from routine infrastructure management tasks.
+
+**Machine learning Lifecycle**
+
+![ML Life cycle](/img/blog_3.png "ML Life cycle")
+
+The ML project life cycle can generally be divided into three main stages: data preparation, model creation, and deployment. All three of these components are essential for creating quality models that will bring added value to your business. This process is *cyclical* because the insights gained from the existing model will help define the next model to be deployed.
+
+In this article, we focus on optimal model identified after data preparation and building model. More specifically, as an example, we shall deploy a sample ONNX model available in Kubernetes native MinIO Object storage, using Triton Inference Server.
```

Comment:
It is preferred, in the HPE Developer blog posts, to not use the "Royal We". Instead, imagine yourself talking directly to the reader. That would mean, in this paragraph, you would say "In this article, I will focus on an optimal model..." (please note some additional words in there).  And the next sentence would say "More specifically, as an example, I will show you how to deploy a sample ONNX model..." (again, a couple of additional words)